torch/onnx/symbolic_opset9.py

```diff
@@ -2207,14 +2207,14 @@ def batch_norm(
 
 def _layer_norm_returns_normalized_input_mean_rstd(
     g,
-    input: torch._C.Value,
-    normalized_shape: Sequence[int],
-    weight: torch._C.Value,
-    bias: torch._C.Value,
+    input: _C.Value,
+    normalized_shape: typing.Sequence[int],
```

Comment:
Import Sequence from typing and use Sequence without the typing namespace. Typing is an exception where we don’t prefix with module name for simplicity

Reply:
https://google.github.io/styleguide/pyguide.html#2241-exemptions

Reply:
Ah. Ok.